### ISSUE TITLE
Query String Trailing Slash

### ISSUE THREAD

**Issue by brayniverse** [FacultyCreative/ngActiveResource#22]
If you append a trailing slash to the url like you do in [this function](https://github.com/FacultyCreative/ngActiveResource/blob/master/lib/angular-resource/base/base.js#L421):

``` javascript
function appendSlashForQueryString(url) {
  if (url.slice(-1) == '/') return url;
  return url + '/';
};
```

It seems to prevent developers from using custom query string parameters like so.

``` javascript

Dashboard.api.showURL = 'http://localhost:8000/api/dashboards/:id/?embed=:embed';

Dashboard.find({
  id: $routeParams.id,
  embed: 'widgets,users'
}).then(function (dashboard) {
  // ...
});
```

Which results in the following `http://localhost:8000/api/dashboards/1/?embed=widgets,users/`

I can't speak for others but I sometimes need to pass query string parameters to my server, maybe to paginate results or to define what resources to embed. If the trailing slash isn't necessary I can send a pull request your way.

**Comment by brettshollenberger** [comment]
Look into it; I think the reason it was added in the first place is because the `$http` method doesn't automatically add the slash before parsing to query string, and I found the slash-then-querystring to be more common than not. Thanks for all your hard work on this, @brayniverse

**Comment by brettshollenberger** [comment]
@brayniverse, this change is fixed on 0.7.3
